### PR TITLE
Reformat fleet API

### PR DESCRIPTION
**Pull request by seiriosPlus**
like title

### CODE REVIEW

File: python/paddle/fluid/incubate/fleet/base/fleet_base.py [hunk 1]
```diff
@@ -15,18 +15,21 @@
 from __future__ import print_function
 
 import abc
-import sys
 
 from enum import Enum
 
 from paddle.fluid.optimizer import SGD
+from paddle.fluid.executor import Executor
```
Comment (guru4elephant) [inline python/paddle/fluid/incubate/fleet/base/fleet_base.py:22]: why do you import Executor here?

File: python/paddle/fluid/incubate/fleet/parameter_server/pslib/__init__.py [hunk 2]
```diff
@@ -28,63 +28,56 @@ class PSLib(Fleet):
...
         init_worker(): will be called by user. When a user knows current process is_server(), he/she
                     should call init_worker() to initialize global information about worker and connect
                     worker with pserver. You should run startup program before init_worker.
 
         Args:
-            programs(Program|list): a Program or a list of Programs
-            scopes(Scope|list): a Scope or  a list of Scopes, default None.
+            executor(Executor): The executor to run for init server.
+            programs(Program|None): The program that need to run.
         """
```
Comment (xjqbest) [inline python/paddle/fluid/incubate/fleet/parameter_server/pslib/__init__.py:49]: what if a user uses  more than one scope , each for a specific main_program.  
in this case,  init_worker no longer supports this.

Comment (xjqbest) [reply python/paddle/fluid/incubate/fleet/parameter_server/pslib/__init__.py:49]: oh，these scopes are passed to fleet in minimize，ignore the above.

File: python/paddle/fluid/incubate/fleet/base/fleet_base.py (hunk 1 above)
Comment (seiriosPlus) [reply python/paddle/fluid/incubate/fleet/base/fleet_base.py:22]: for value check, `isinstance of`

**Review by guru4elephant — APPROVED** [review]
LGTM. You need to update official document with replicable examples and API docs.

**Review by fuyinno4 — APPROVED** [review]

**Pull request merged**